src/components/Popup/positioningHelper.ts

```diff
@@ -74,3 +74,30 @@ export default ({
 
   return `${computedPosition}${stringifiedAlignment}` as Placement
 }
+
+/////////////////////////////////
+// OFFSET VALUES ADJUSTMENT
+/////////////////////////////////
+
+const flipPlusMinusSigns = (offset: string): string => {
+  return offset
+    .replace(/\-/g, '<plus>')
+    .replace(/^(\s*)(?=\d)/, '<minus>')
+    .replace(/\+/g, '<minus>')
+    .replace(/<plus>/g, '+')
+    .replace(/<minus>/g, '-')
+    .trimLeft()
+    .replace(/^\+/, '')
+}
```

Comment:
This looks a complicated for me, but we have tests. Possibly, we can to add memoization there.

Reply:
yes, the tests were introduced exactly for this sake :) not sure about memoization, as all expressions used are of linear complexity - there shouldn't be any significant gain from it.